docs/mp/grpc/server-side-services.adoc

```diff
@@ -18,16 +18,24 @@
 
 = gRPC MicroProfile Server Services
 :description: Helidon gRPC MicroProfile Server-Side Services
-:keywords: helidon, grpc, microprofile, micro-profile
+:keywords: helidon, java, grpc, microprofile, micro-profile, mp
 :feature-name: gRPC MicroProfile Server
 :rootdir: {docdir}/../..
 :microprofile-bundle: false
 
 include::{rootdir}/includes/mp.adoc[]
 
+== Contents
+
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<API, API>>
+- <<Usage, Usage>>
+
+== Overview
 The gRPC Microprofile APIs are an extension to xref:../introduction/introduction.adoc[Helidon MP] to allow building
```

Comment:
check link

Reply:
This link was already replaced with `xref:{rootdir}/mp/introduction.adoc[Helidon MP]` in PR https://github.com/oracle/helidon/pull/4606,

Reply:
Updated